config/settings.yml

```diff
@@ -82,3 +82,12 @@ plugins:
     default: ''
   max_translations_per_minute:
     default: 3
+  restrict_translation_by_group:
+    default: ''
+    client: true
+    type: list
+    choices: # Valid choices retrieved from user.groups
+      - admins
+      - moderator
+      - staff
+      - '' # Same as everyone
```

Comment:
Hmm, can we use "everyone explicitly here and in the default value? An empty string doesn't look right to me (not sure if we use empty strings elsewhere to indicate "everyone", but I think it's best to be explicit).

Reply:
Ah, on second thought, the list of choices here doesn't include all groups. Usually with `_groups` settings, we want the admins to be able to pick any group. 

I think you can use `type: group_list` here, and that will automatically use a group dropdown. And then for the default, you can default to `trust_level_1`, like we do in core, for example here: 

https://github.com/discourse/discourse/blob/main/config/site_settings.yml#L837-L840